python/az/aro/azext_aro/_params.py

```diff
@@ -113,6 +114,12 @@ def load_arguments(self, _):
         c.argument('worker_subnet',
                    help='Name or ID of worker vnet subnet.  If name is supplied, `--vnet` must be supplied.',
                    validator=validate_subnet('worker_subnet'))
+        c.argument('load_balancer_managed_outbound_ip_count',
+                   type=int,
```

Comment:
Comment: Upstream specifically wanted int32 formats here. Should work just fine but just calling it out.

Reply:
I think that Tanmay had mentioned that we should likely hold on this until we do the downstreaming of all the REST Specs work, then rebase this PR on that.

Reply:
A quick peek at AKS's equivalent flag shows they've defined the type as `int` as well: https://github.com/Azure/azure-cli/blob/d419ab357481e803d2f2a84a40760fa6e6af7c8b/src/azure-cli/azure/cli/command_modules/acs/_params.py#L334

I'm not sure if the API specificity on the data type necessarily correlates to the CLI/Python-side requirements here.

Reply:
native python3 doesn't have a int32 type

Reply:
are we good to resolve this?